### ISSUE TITLE
Pkg.build fails on Windows 10

### ISSUE THREAD

**Issue by burdickjp** [JuliaPackaging/WinRPM.jl#114]
`Julia Version 0.6.0
Commit 903644385b* (2017-06-19 13:05 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-2640M CPU @ 2.80GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Sandybridge)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, sandybridge)`

`LoadError: "ErrorException(\"Error parsing document : 0\"), not well-formed (invalid token), 1, 9865, 9865"
while loading C:\Users\Jeffrey.Burdick\.julia\v0.6\WinRPM\deps\build.jl, in expression starting on line 2`

[I've tried this](https://github.com/JuliaPackaging/WinRPM.jl/issues/104). It didn't work.

**Comment by musm** [comment]
@burdickjp  Could you please try https://github.com/musm/WinRPM.jl  ?

**Comment by robertdj** [comment]
After upgrading to Julia v0.6 I'm experiencing the problem again and this time my "trick" did not work.

@musm : I've tried cloning your fork, but I'm getting the same parsing error as above.

**Comment by robertdj** [comment]
It looks like like your pull requests have been reviewed and approved, but not accepted?
If they fix the issues I would *love* for them to be included!
@ararslan @tkelman

**Comment by musm** [comment]
@robertdj  yes if you urgently need a fix please checkout https://github.com/JuliaPackaging/WinRPM.jl/pull/121 and let me know if it works?

**Comment by robertdj** [comment]
I'll try -- thanks!

**Comment by musm** [comment]
Ops linked wrong pr above

https://github.com/JuliaPackaging/WinRPM.jl/pull/122

**Comment by robertdj** [comment]
I **think** I managed to merge the right commit (6198d00aea140a3d8f248f5bf42ffca6c927a90b), but when I run `Pkg.build("WinRPM")` I still get the same error.

**Comment by Gyslain** [comment]
I'm having the same issue. My julia versioninfo() reports : 
```Julia Version 0.6.1
Commit 0d7248e2ff* (2017-10-24 22:15 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i5-3320M CPU @ 2.60GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Sandybridge)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, ivybridge)```
